src/commands/build.ts

```diff
@@ -32,6 +32,21 @@ export class BuildCommand implements Command {
 
   args = [
     {
+      name: 'js-minify',
```

Comment:
I don't think it's that uncommon to use `.` when flags map to JSON-like structures. Mapping with `-` is harder: does `foo-bar-baz` map to `foo.bar.baz`, `foo.bar-baz`, etc.